website/content/docs/agent/telemetry.mdx

```diff
@@ -595,7 +595,30 @@ These metrics are used to monitor the health of the Consul servers.
 ** Requirements: **
 * Consul 1.12.0+
 
-Label based RPC metrics were added in Consul 1.12.0 as a Beta feature to better understand the workload on a Consul server and, where that workload is coming from. The following metric(s) provide that insight
+Label based RPC metrics were added in Consul 1.12.0 as a Beta feature to better understand the workload on a Consul server and, where that workload is coming from. The following metric(s) provide that insight.
+
+All RPC metric method calls are by default blocked via the [`prefix_filter`](/consul/docs/agent/config/config-files#telemetry-prefix_filter) telemetry configuration setting. You can enable all RPC metric server
+calls and/or explicity deny RPC calls using the same configuration setting.
```

Comment:
```suggestion
The [`prefix_filter`](/consul/docs/agent/config/config-files#telemetry-prefix_filter) telemetry configuration setting blocks or enables all RPC metric method calls. Specify the RPC metrics you want to allow in the `prefix_filter`:
```
To be consistent with our content types guidance, we should really put this in the agent configuration reference and to it from here.